pkg/cmd/connector/addon/parameters.go

```diff
@@ -0,0 +1,99 @@
+package addon
+
+import (
+	"github.com/redhat-developer/app-services-cli/pkg/cmd/connector/connectorcmdutil"
+	"github.com/redhat-developer/app-services-cli/pkg/core/cmdutil/flagutil"
+	"github.com/redhat-developer/app-services-cli/pkg/core/ioutil/dump"
+	"github.com/redhat-developer/app-services-cli/pkg/shared/connection"
+	"github.com/redhat-developer/app-services-cli/pkg/shared/factory"
+	connectormgmtclient "github.com/redhat-developer/app-services-sdk-go/connectormgmt/apiv1/client"
+
+	"github.com/spf13/cobra"
+)
+
+// row is the details of the addon parameters needed to print to a table
+type itemRow struct {
+	ID    string `json:"id" header:"ID"`
+	Value string `json:"value" header:"Value"`
+}
+
+type options struct {
+	outputFormat string
+	id           string
+
+	f *factory.Factory
+}
+
+func NewParametersCommand(f *factory.Factory) *cobra.Command {
+	opts := &options{
+		f: f,
+	}
+
+	cmd := &cobra.Command{
+		Use:     "addon-parameters",
+		Short:   f.Localizer.MustLocalize("connector.addon.params.cmd.shortDescription"),
+		Long:    f.Localizer.MustLocalize("connector.addon.params.cmd.longDescription"),
+		Example: f.Localizer.MustLocalize("connector.addon.params.cmd.example"),
+		Args:    cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if opts.outputFormat != "" && !flagutil.IsValidInput(opts.outputFormat, flagutil.ValidOutputFormats...) {
+				return flagutil.InvalidValueError("output", opts.outputFormat, flagutil.ValidOutputFormats...)
+			}
+
+			return runList(opts)
+		},
+	}
+
+	flags := connectorcmdutil.NewFlagSet(cmd, f)
+
+	flags.AddOutput(&opts.outputFormat)
+	_ = flags.AddConnectorID(&opts.id).Required()
+
+	return cmd
+}
+
+func runList(opts *options) error {
+	f := opts.f
+	conn, err := f.Connection(connection.DefaultConfigSkipMasAuth)
+	if err != nil {
+		return err
+	}
+
+	api := conn.API()
+
+	a := api.ConnectorsMgmt().ConnectorClustersApi.GetConnectorClusterAddonParameters(f.Context, opts.id)
```

Comment:
Looks like `addon-parameters` should be a sub-command of `connector cluster` rather than `connector`.
Any reason it has been placed here @wtrocki ?

Reply:
Actually I was thinking about it. Yes they should be in cluster.